test/testing.cmake

```diff
@@ -526,6 +526,14 @@ mpi_test(l2_shape_tet_parallel 4
   ./L2Shapes
   ".null"
   "${MDIR}/4/cube.smb")
+mpi_test(h1_shape_serial 1
+  ./H1Shapes
+  ".null"
```

Comment:
any reason not to load the model? (same for the following test)

Reply:
These test only depend on the mesh. I have added the models to keep things uniform with the surrounding code.